src/components/AsideSection/AsideSection.tsx

```diff
@@ -24,10 +25,9 @@ export interface AsideSectionProps extends NestableBaseComponentProps {
 /**
  * A section component to be used primarily for sidebar modules
  */
-@customizable('AsideSection', ['theme'])
-export default class AsideSection extends React.Component<AsideSectionProps & CustomizableComponentProps> {
+export class AsideSection extends React.Component<AsideSectionProps & CustomizableComponentProps> {
```

Comment:
This breaks the pattern a bit, since `AsideSectionProps` extends `NestableBaseComponentProps` instead of `BaseComponent` props.

Not sure if there are tradeoffs between that and using intersection like this.